src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,116 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+interface Params {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+}
+
+function checkCurrentPage(page: number, toCheckValue: number): boolean {
+  return page === toCheckValue;
+}
+
+export const Pagination: React.FC<Params> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pages = Math.ceil(total / perPage);
+  const pagesArr: number[] = [];
+  let items: number[] = [];
```

Comment:
you already have items implemented. in the to of App file. use them and split properly for pages. instead of creating again